src/numa_threadpool.rs

```diff
@@ -49,7 +49,7 @@ pub use self::numa::*;
 mod numa {
     extern crate nix;
 
-    use libc::{c_char, c_uint, c_int, c_ulong};
+    use numa_threadpool::numa::nix::libc::{c_char, c_uint, c_int, c_ulong};
```

Comment:
I don't quite understand why removing the unstable flag caused the build to fail for missing imports, the compile suggested this and it fixed it.

Reply:
That's because lib.rs has a `#[cfg(feature="unstable")]` mark on the libc extern crate (and I think cargo conditional-depends on it as well)

Reply:
To fix this:

1. Remove `optional=true` from the libc clause in Cargo.toml
2. Remove "libc" from the feature definitions in Cargo.toml
3. Add an `extern crate libc;` to src/lib.rs - or update to Rust 2018 :)